wiki/Skinning_Tutorial/en.md

```diff
@@ -0,0 +1,41 @@
+# Skinning Tutorial
+
+*See also: [Skinning](/wiki/Skinning).*
+
+*This article is the tutorial for skinning. For frequently answered skinning questions, see [Skinning/FAQ](/wiki/Skinning/FAQ).*
+
+## Getting Started
+
+To get started, open the game client, select [Options](/wiki/Options), and click `Open osu! folder`. Once the file explorer opens, click on `Skins/` and create a new folder. You can name it whatever you want, but keep in mind that osu! will use this name when it is displayed in the options. Inside this folder, you can add any skinning elements listed in the skin element lists.
```

Comment:
`file explorer` -> `directory`?

`osu! will use this name when it is displayed in the options` -> `osu! will display this name in the options`